### DISCUSSION TITLE
How to find field accessors?

### DISCUSSION THREAD

**Discussion by JulianWinter1124** [vfsfitvnm/frida-il2cpp-bridge#80] [Q&A]
Hey! I want to observe a specific field in a class. How do I achieve that?
Some crude example code:
```
class Counter : System.Object
{
    System.Int64 count;
}

//hidenn somewhere:
class CounterSetter : System.Object
{
   System.Void someMethod(Counter counter);
}
```
Let's say Counter doesnt explicitely define a setter method so the field is accessed directly by other classes. 
Now whenever some other class accesses count I want to know where that call came from (and the value).
Thanks for the help.

**Comment by vfsfitvnm** [comment]
Hi! There's no such API for that, yet. However, you could try [MemoryAccessMonitor](https://frida.re/docs/javascript-api/#memoryaccessmonitor)

**Reply by vfsfitvnm** [reply]
However, there's no elegant way to obtain the `Il2Cpp.Method` who accessed that field, just the raw NativePointer

**Reply by vfsfitvnm** [reply]
Hmm, just tried that and couldn't make it work. I get access violation errors for no apparent reason

**Reply by JulianWinter1124** [reply]
Thank you for reaching back and trying, I didn't have the time yet to get into the `MemoryAcessMonitor` myself. Still on a very beginner level  in Frida's API.
